python_modules/dagster/dagster/core/scheduler/instigation.py

```diff
@@ -55,6 +55,7 @@ def __new__(cls, cron_schedule, start_timestamp=None, scheduler=None):
             # `start_date` on partition-based schedules, which is used to define
             # the range of partitions)
             check.opt_float_param(start_timestamp, "start_timestamp"),
+            # this is a vestigial parameter that is not used and will be removed in the future
```

Comment:
is there a reason not to remove the param now? I think the serdes will still work?